__tests__/app.tests.js

```diff
@@ -438,4 +444,30 @@ describe("COMMENTS TESTS", () => {
       );
     });
   });
+
+  describe("DELETE /api/comments/:comment_id", () => {
+    test("Returns 204 status code for a valid comment id that is in the database", async () => {
```

Comment:
Why are you testing for a comment that exists if you're looking at the DELETE method in this block?

Also, the 204 status code would be incorrect for a successful GET request.

Reply:
The 204 is for the delete DELETE.
The GET requests are to check articles endpoint, the first checks if there is no longer a comment with that id, and the second checks the count of comments after deletion.